collections/_posts/2023-07-10-self-explaining-neural-networks.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: review
+title: "Towards Robust Interpretability with Self-Explaining Neural Networks"
+tags: interpretability deep-learning intrinsic
+author: "Pierre-Elliott Thiboud"
+cite:
+    authors: "David Alvarez-Mellis, Tommi S. Jaakkola"
+    title:   "Towards Robust Interpretability with Self-Explaining Neural Networks"
+    venue:   "NeurIPS 2018"
+pdf: https://arxiv.org/pdf/1806.07538.pdf
+---
+
+Github available [here](https://github.com/dmelis/SENN) (PyTorch)
+
+# Highlights
+
+- Post-hoc interpretability methods are not always faithful in regard to model's predictions and their explanations aren't robust to small perturbations of the input[^1]
+- The authors propose a class of intrinsically interpretable neural networks models which are as interpretable as linear regression
+
+# Introduction
+
+- Most interpretability methods are post-hoc, explaining the model after its training
+- But their explanations lack stability, small changes to the input can lead to different (or even contradicting) explanations for similar predictions
```

Comment:
_Contradicting_ and _contradictory_ are often mixed up, but they're not interchangeable. In your case you mean to use _contradictory_. See [this question](https://english.stackexchange.com/questions/409244/difference-between-contradictory-and-contradicting) for the differences between the two.
```suggestion
- But their explanations lack stability, small changes to the input can lead to different (or even contradictory) explanations for similar predictions
```